yo/app/views/directives/filter-selector.html

```diff
@@ -6,6 +6,8 @@
 
             <button class="btn btn-danger pull-right" ng-click="removeFilter()"><i class="icon icon-remove"></i></button>
 
+            <button class="btn btn-danger pull-right" ng-click="removeFilter()"><i class="icon icon-remove"></i></button>
```

Comment:
yepp, was probably a c'n'p or merging issue. I've fixed this in a later commit, which is on top of this state.
If you wish that I should fix here as well, I can do so. Otherwise, I can just say that it is already fixed, but in a more recent revision.